apps/web/frontend/components/calendarSelector.tsx

```diff
@@ -101,11 +102,12 @@ export const CalendarSelector = (props: CalendarSelectorProps): JSX.Element => {
         <Popover.Button
           ref={reference}
           aria-label="select date"
-          className="flex disabled:opacity-50"
+          className="flex gap-1 disabled:opacity-50"
           disabled={props.disabled ?? false}
         >
           {!props.hideLabel && <span title="Display value">{props.date?.toLocaleDateString()}</span>}
-          <AiOutlineCalendar className="ml-2" size="1.3em" title="Calendar icon" />
+          <AiOutlineCalendar className="ml-2 h-5 w-5" title="Calendar icon" />
```

Comment:
The parent already has `gap-1`.

```suggestion
          <AiOutlineCalendar className="h-5 w-5" title="Calendar icon" />
```